src/sqlfluff/rules/L019.py

```diff
@@ -191,9 +191,9 @@ def _eval(self, segment, raw_stack, memory, **kwargs):
                         fixes=[
                             LintFix("delete", anchor=last_comma_seg),
                             LintFix(
-                                "edit",
+                                "create",
                                 anchor=segment,
-                                edit=[last_comma_seg, new_whitespace_seg, segment],
```

Comment:
Editing `segment` where the replacement **includes** segment` causes unpredictable behavior. I've seen this in other rules previously.